### DISCUSSION TITLE
Documentation please

### DISCUSSION THREAD

**Discussion by plungu** [postmanlabs/postman-flows#257] [General]
Things changing suddenly with no documentation or examples. Classic blunder! 
A tool tip on the blocks would be really great. I'm having a lot of difficulty figuring out how some of the blocks work. 
If you want fast adoption please make it usable by actually documenting how things are supposed to work.

**Comment by saswatds** [comment]
Hey @plungu we are currently maintaining the document here https://www.postmanlabs.com/postman-flows/ . Also any changes we are making to flow is being added to the release note section here on Github https://github.com/postmanlabs/postman-flows/releases. 

We will continue making the documentation easier to consume with more examples.

**Reply by alexei-moneyr** [reply]
This is not documentation. 
Is is super short and bad described md. Please add some more to this. 
Tool is great, but hard to work with
